examples/llm_ptq/scripts/huggingface_example.sh

```
@@ -227,37 +193,16 @@
-            $BUILD_ARGS
-    else
-
-        if [[ ! " fp8 nvfp4 bf16 fp16 " =~ " ${QFORMAT} " ]]; then
-            echo "Quant $QFORMAT not supported with the TensorRT-LLM torch llmapi. Allowed values are: fp8, nvfp4, bf16, fp16"
-            exit 0
-        fi
-
-        if $TRUST_REMOTE_CODE; then
-            RUN_ARGS+=" --trust_remote_code "
-        fi
+    if [[ ! " fp8 nvfp4 bf16 fp16 int4_awq w4a8_awq " =~ " ${QFORMAT} " ]]; then
```

Shellcheck (0.10.0)

[warning] Remove quotes from right-hand side of =~ to match as a regex rather than literally.

(SC2076)